modules/router/templates/draft-assets.conf.erb

```diff
@@ -50,7 +50,7 @@ server {
     return 404;
   }
   <%- if scope.lookupvar('::aws_migration') %>
-  set $upstream_asset_manager <%= @upstream_ssl ? 'https' : 'http' %>://asset-manager.<%= @app_domain %>;
+  set $upstream_asset_manager <%= @upstream_ssl ? 'https' : 'http' %>://draft-assets-carrenza.<%= @app_domain %>;
```

Comment:
I think leaving app_domain here will result in draft-assets-carrenza.production.govuk-internal.digital - not sure if we have set app_domain_external to publishing.service.gov.uk in this context - that might be ideal

Reply:
Yes, good point. By using app_domain instead of app_domain_internal we should fix that